### PR TITLE
new splash screen design

### PR DESCRIPTION
**Pull request by talwag**

### CODE REVIEW

**Comment by talwag** [comment]
image

File: packages/donor/src/screens/authentication/AuthLoadingScreen.module.scss [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 }
+// @keyframes pulsate {
+//   0% {
+//     opacity: 0.8;
+//   }
+//   50% {
+//     opacity: 0;
+//   }
+//   100% {
+//     opacity: 0.8;
```
Comment (yaron1m) [inline packages/donor/src/screens/authentication/AuthLoadingScreen.module.scss:20]: Please remove

**Review by yaron1m — APPROVED** [review]
Very nice!

File: packages/donor/src/screens/authentication/AuthLoadingScreen.module.scss (hunk 1 above)
Comment (talwag) [reply packages/donor/src/screens/authentication/AuthLoadingScreen.module.scss:20]: done

**Pull request merged**